librz/bin/p/bin_elf.inc

```diff
@@ -1389,33 +1389,34 @@ static RzBinReloc *reloc_convert(ELFOBJ *bin, RzBinElfReloc *rel, ut64 GOT) {
 		r->section_vaddr = section->rva;
 	}
 
-#define SET(T) \
+#define SET(T, N) \
```

Comment:
```suggestion
#define SET(T, RELOC_NAME) \
```

Nitpicky, though it saves one more lookup.
Same below

Reply:
You're right, changing it now